security/voters.rst

```diff
@@ -278,12 +278,16 @@ There are three strategies available:
     This grants access as soon as there is *one* voter granting access;
 
 ``consensus``
-    This grants access if there are more voters granting access than denying;
+    This grants access if there are more voters granting access than
+    denying. If case of a tie the decision is based on the
```

Comment:
Is it correct to say `If case of` and  not `In case of`?

Reply:
fixed in c4dd58475